init-tools.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+# Load Branch Info
+while read line; do
+    if [[ $line != \#* ]]; then
+        IFS='=' read -ra splat <<< "$line"
+        export ${splat[0]}="${splat[1]}"
+    fi
+done < "$DIR/branchinfo.txt"
+
+$DIR/scripts/obtain/dotnet-install.sh --channel $CHANNEL --verbose
```

Comment:
There's some things that happen in init-tools.ps1 that aren't happening here.  Can these files be consistent?